lib/string_decoder.js

```diff
@@ -58,8 +58,8 @@ const StringDecoder = exports.StringDecoder = function(encoding) {
 // returned when calling write again with the remaining bytes.
 //
 // Note: Converting a Buffer containing an orphan surrogate to a String
-// currently works, but converting a String to a Buffer (via `new Buffer`, or
-// Buffer#write) will replace incomplete surrogates with the unicode
+// currently works, but converting a String to a Buffer (via `Buffer.from()`,
```

Comment:
`Buffer#write` is used as a shortcut for `Buffer.prototype.write`, so it’s something you can call on an `Buffer` instance. `Buffer.from()` however needs to be called on the global `Buffer` object (just as it is written). So, no, it is correct to leave these as they are.